docs/administration/segregation.md

```diff
@@ -1,7 +1,118 @@
 # Data segregation
 
-!!! tip "Under construction"
+## Introduction
+
+*Data segregation* in the context of *Cyber Threat Intelligence* refers to the practice of categorizing and separating different types of data or information related to cybersecurity threats based on specific criteria.
+
+This separation helps organizations manage and analyze threat intelligence more effectively and securely and the goal of data segregation is to ensure that only those individuals who are authorized to view a particular set of data have access to that set of data.
+
+## Marking Definitions
+
+### Description
+
+*Marking definitions* are essential in the context of data segregation to ensure that data is appropriately categorized and protected based on its sensitivity or classification level. Marking definitions establish a standardized framework for classifying data.
+
+Marking Definition objects are unique among STIX objects in the [STIX 2.1 standard](https://docs.oasis-open.org/cti/stix/v2.1/stix-v2.1.html) in that they cannot be versioned.  This restriction is in place to prevent the possibility of indirect alterations to the markings associated with a STIX Object.
+
+Multiple markings can be added to the same object, including both object and granular markings. Certain categories of marking definitions or trust groups may enforce rules that specify which markings take precedence over others or how some markings can be added to complement existing ones.
+
+In OpenCTI, data is segregated based on knowledge marking. The diagram provided below illustrates the manner in which OpenCTI establishes connections between pieces of information to authorize data access for a user:
+
+![Data access diagram](assets/data-access-diagram.png)
+
+### Traffic Light Protocol
+
+The *Traffic Light Protocol* (TLP) was designed by the *Forum of Incidence Response and Security Teams* ([FIRST](https://www.first.org/tlp/)) to provide a standardized method for classifying and handling sensitive information, based on four categories of sensitivity.
+
+For more details, the diagram provided below illustrates how are categorized the marking definitions:
+
+![TLP diagram](assets/tlp-diagram.png)
+
+### Create new markings
+
+In order to create a marking, you must first have the ability to access the Settings tab. For example, a user who is in a group with the role of Administrator can bypass all capabilities or a user who is in a group with the role that has `Access administration` checked can access the Settings tab.
+
```

Comment:
you can reference here the doc page on users and rbac (../administration/users.md) I think